en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-api-manager.md

```diff
@@ -0,0 +1,320 @@
+# Upgrading WSO2 API Manager
+
+This document walks you through the process of upgrading WSO2 API Manager. 
+
+## Why migrate?
+
+There are multiple reasons why you would want to upgrade the WSO2 product to the latest version. These reasons include but are not limited to the following.  
+
+- The current product version you are using is reaching its end of life. To see if this is the case, view the [support matrix documentation](https://wso2.com/products/support-matrix/). 
+- You want to leverage the new features of the latest version of the product.
+- The version of the product you have does not have certain security and bug fixes that you require.
+
+## What has changed
+
+Over the course of its lifetime, WSO2 API Manager has changed significantly and some of the features you were using in an older version may not work the same way. This section summarizes the key changes that have taken place in each version of WSO2 APi Manager.
+
+??? note "API Manager 4.1.0"
+
+    ##### Mediation policies migration
+
+    With the 4.1.0 release, WSO2 API Manager has realigned its previous API-level mediation policies feature to a more sophisticated policy feature that provides support for not only mediation policies, but a vast number of different use cases. 
+    
+    Instead of API-level policy allocation, the new feature supports policies at the operation level for the regular Gateway (Synapse-based) and at resource level for the Choreo Connect Gateway. With this support, previous mediation policy support via Admin REST APIs is no longer available. Instead Publisher REST API `operation-policies` support the same functionality. Publisher UI mediation policies, which were displayed under the runtime configuration, have been removed and it is moved to a separate tab called `policies`. Therefore, you will be able to see the mediation policies in previous API-M versions under the policies UI.
```

Comment:
```suggestion
    Instead of API-level policy allocation, the new feature supports policies at the operation level for the regular Gateway (Synapse-based) and at resource level for the Choreo Connect Gateway. With this support, previous mediation policy support via Admin REST APIs is no longer available. Instead, Publisher REST API `operation policies` support the same functionality. Publisher UI mediation policies, which were displayed under the runtime configuration, have been removed, and it is moved to a separate tab called `policies`. Therefore, you will be able to see the mediation policies in previous API-M versions under the policies UI.
```